script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -739,50 +750,51 @@ A grim foreboding crawls up my spine, and I my heart gives off a loud pulsation.
 -- 前に話したでしょ、親基と下僕の関係。死者はみんな死体に戻っただろうけど、死徒にまで成ったのが数体残ってるみたいなの。
 I told you about this before, right?
  How sires and underlings are connected.
- The Dead have probably all went back to being mere cadavers,
- but it seems there are a few who managed to climb all the way to Dead Apostle still remaining.
+ The Dead have probably all gone back to being mere cadavers,
+ but it seems a couple of his underlings climbed far enough in the ranks to become Dead Apostles in their own right,
+ and are still hanging around.
 }
 [sha:e5e262e6daf6b4d5a6616228447b0e386f9a4f20]{
 -- Page 486, Offset 15964. Glued. Mods: @x.
 -- それでね、志貴」
-So with that said, Shiki."
+ So with that said, Shiki..."
 }
 [sha:7afc659f200416795db5c1ef27457a10c0ee049f]{
 -- Page 487, Offset 15965.
 -- アルクェイドは甘えた声で俺の名前を口にした。
-Arcueid calls my name with an almost sweet tenor.
+Arcueid calls my name in a wheedling tone.
 }
 [sha:d31b059ab9c682251682c75ea51fef62e4335238]{
 -- Page 487, Offset 15966.
 -- ……読めてきた。
-I saw this coming.
+...I can see where this is going.
 }
 [sha:aae64f95f55e6fb58a0ca7f4610b8a38b7ba0e23]{
 -- Page 487, Offset 15967.
 -- というか、読めてしまった。
 // well, well, しまった my old enemy.
-I hate that I saw it coming.
+I hate that I can see where this is going.
 }
 [sha:abe974b8a65a8ddd9de1e2d998bd698a77b5b6a2]{
 -- Page 487, Offset 15968.
 -- アルクェイドが俺を呼びつけたのは、つまり、
-Basically, the reason Arcueid called me out here is――
+Basically, the reason Arcueid called me out here is because―――
 }
 [sha:fd0499dc067d6e87fb31d4166182dbfddf5bd931]{
 -- Page 488, Offset 15969.
 -- 「ロアを倒すのに力を使っちゃったから疲れちゃって。
-"I'm tired from having to use all that power to defeat Roa.
+"I ended up using a bunch of my strength to defeat Roa, so I'm a little worn out.
 }
 [sha:509adc7cd20b12744f0ff7ca7788b7916d95d9f2]{
 -- Page 488, Offset 15970.
 -- 誰かがわたしと一緒に、死徒の後始末をしてくれたら助かるんだけどなあ」
-Oh, if only someone could take care of these Dead Apostles with together with me, I would sooo appreciate it."
+Oh, if only there were %{i}someone%{/i} out there who'd come to my aid and help me take care of these Dead Apostles..."
```

Comment:
I think the previous version is both more accurate and reads better.

Reply:
not sure I agree (ignoring the typo). I prefer the new version personally.